### ISSUE TITLE
Cannot enable "show hidden files" in file manager.

### ISSUE THREAD

**Issue by Lend27** [agajews/ApricityOS#21]
Once "show hidden files" is selected, the file manager freezes for a few seconds and then disappears taking the desktop icons with it. You can click on "file manager" again to bring the icons back but it will happen every time hidden files is clicked.

**Comment by agajews** [comment]
That's odd. It could be something specific for your hardware. What kind of machine do you have?

**Comment by Lend27** [comment]
I am running a Gigabyte Motherboard with an i7-4790k processor and 32 gig of ram.
Nvidia GTX-760 graphics card. I installed Thunar and no problems with it. The problem is only with the default "Files" program.

**Comment by gogobebe2** [comment]
I had this same problem. Could also be reproduced on another Arch based distro called Antergos.

**Comment by agajews** [comment]
Do you also have Nvidia graphics?

**Comment by gogobebe2** [comment]
Yes. I also had problems with screen resolution which made me give up on getting ApricityOS to work so I've been waiting for October beta release.

**Comment by agajews** [comment]
So with the October Beta we're enabling bumblebee by default, which may change things. It will hopefully be released this weekend, depending on how long it takes for us to make a dark shell theme.

**Comment by gogobebe2** [comment]
Ok, looking forward to it.

**Comment by gogobebe2** [comment]
Sorry, where can I see what the latest version is and perhaps get updates of when it new ones come out?

**Comment by agajews** [comment]
We have a release notes blog that's linked on the Apricity site and has an RSS feed: http://apricityos.com/blog/index.php/feed/

**Comment by gogobebe2** [comment]
Thanks

**Comment by agajews** [comment]
It's also possible that this was fixed in Gnome 3.18. Could one of you test this?

**Comment by gogobebe2** [comment]
Sure, I'll test it with the live CD. So should I just download ApricityOS from the website and use that build?

**Comment by agajews** [comment]
The new iso isn't quite live yet. If you still have an old one, or want to re-download the old one, you can update the packages in it and reload Gnome from the alt-F2 looking glass by entering r.

**Comment by agajews** [comment]
Alternatively, you can wait until tomorrow morning and download the new iso then.

**Comment by gogobebe2** [comment]
I'll just wait because I might actually install it too.

**Comment by agajews** [comment]
Yup

**Comment by gogobebe2** [comment]
Ok, thanks agajews. Looking forward to it.

**Comment by Murii** [comment]
Off topic:
1)If I want to update to the newest version of the OS which will come tomorrow do I need to reinstall the whole OS or I can just update it from somewhere?
2)Right now I have around 300 updates. I tried to upgrade all the software that is required but when I started the OS it freezed at the loading point. So I needed to reinstall it and this time I did not upgrade anything. Am I the only one with this problem?

**Comment by agajews** [comment]
1. The only things that are new are things that can be upgraded by updating all packages in the system. 
2. Actually one other person documented something similar. What kind of machine do you have? We can try to draw connections between the two and see what similarities there are. 
3. This update makes Apricity OS one of the very first operating systems to ship with version 3.18 of Gnome, a release that improves touch compatibility, makes the file manager more powerful and intuitive, and fixes a bunch of bugs, to name a few key changes. Additionally, Apricity OS now has better support for Nvidia graphics cards, a tweaked shell theme, and several bug fixes.

**Comment by Murii** [comment]
My PC is an laptop from acer which has: cpu - i3 1.7 , gpu - 1 gb vram from nvidia 820m and 4 gb ram ddr3

**Comment by Murii** [comment]
I would say that the problem with the update was that some packages were trying to ugrade to gnome 3.16 -> 3.18.
 I 'll wait for further instructions.

**Comment by agajews** [comment]
The other person that had that problem actually ran his update the day before Gnome 3.18 was pushed to stable repositories, so it's probably not that. He also had Nvidia graphics, so it's more likely that it's an issue with the drivers. Which proprietary drivers (if any) did you install?

**Comment by Murii** [comment]
I'm not sure which driver version I have but I know that I have them installed because I'm also too a programmer and I'm using opengl so I really need drivers. Also I don't think it's related to nvidia because I used to use Antergos OS  with OpenBox and there was no problems updating.

**Comment by agajews** [comment]
It could be an Nvidia+Gnome issue then, because that seems to be the only link between you two. Also, it's extraordinarily unlikely that an update could cause data loss as it's always possible to boot with a live image and have direct access to an unencrypted hard drive. This issue sounds quite a bit like an issue that existed with a faulty Plymouth script that was fixed, so I wonder of it's not Plymouth and Nvidia misbehaving. Did you by any chance install your proprietary Nvidia drivers using the .run files on the Nvidia website?

**Comment by Murii** [comment]
Nah,I have the drivers installed by default

**Comment by Cauterize-Linux** [comment]
Is this still an issue for you guys? I don't have a Nvidia card to test if this is a common root cause.

**Comment by VlentfulGaming** [comment]
I have a Nvidia GTX 970, and it seems to work fine for me. I bet it is fixed as its an issue from 8 months ago. I am loving ApricityOS so far.

**Comment by gogobebe2** [comment]
Yep, I don't have this issue anymore :+1:
